addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -373,17 +373,29 @@ def _import_partner(self, company_id, name, phone, email, vat, country_code=Fals
         partner = self.env['res.partner'] \
             .with_company(company_id) \
             ._retrieve_partner(name=name, phone=phone, email=email, vat=vat, domain=domain)
+        country_code = postal_address.get('country_code')
+        country = self.env.ref(f'base.{country_code.lower()}', raise_if_not_found=False) if country_code else self.env['res.country']
+        state_code = postal_address.get('state_code')
+        state = self.env['res.country.state'].search(
```

Comment:
Please add a limit=1 for extra safety here. Shouldn't happen to have two states for a single country with the same code but heh.